gradio/components/image.py

```diff
@@ -229,10 +251,18 @@ def postprocess(
         if value is None:
             return None
         if isinstance(value, str) and value.lower().endswith(".svg"):
-            return FileData(path=value, orig_name=Path(value).name)
+            return ImageData(path=value, orig_name=Path(value).name)
+        if self.format == "base64":
+            if isinstance(value, np.ndarray):
+                return ImageData(url=image_utils.encode_image_array_to_base64(value))
```

Comment:
It seems that you don't even use the b64 field, you just reuse the url field. Is that intended? If this works, then I think we should just reuse FileData dataclass instead of creating a new ImageData class